hknweb/academics/models/logistics.py

```diff
@@ -14,7 +19,12 @@ class Department(AcademicEntity):
 
 
 class Instructor(AcademicEntity):
-    instructor_id = models.TextField(max_length=500, primary_key=True)
+    instructor_id = models.TextField(max_length=500)
+    current_first_name = models.TextField(blank=True, null=True)

```

Comment:
Add these to the corresponding Serializer e.g. https://github.com/compserv/hknweb/blob/8707226e264c87e871a050c73134701dae244fa5/hknweb/academics/serializers/logistics_serializers.py#L9